spec/controllers/password_resets_controller_spec.rb

```diff
@@ -0,0 +1,111 @@
+RSpec.describe PasswordResetsController do
+  describe "POST #create" do
+    let(:user)      { create(:confirmed_user) }
+    let(:params)    { { password_reset: { email: user.email } } }
+    let(:arguments) { { email: user.email } }
+    let(:context)   { double(:context, success?: true) }
+
+    before(:example) do
+      allow(PasswordResetOrganizer).to receive(:call).with(arguments).
+        and_return(context)
+    end
+
+    it "calls PasswordResetOrganizer" do
+      expect(PasswordResetOrganizer).to receive(:call)
```

Comment:
@Enriikke why do we have to put the `expect` before we create the http request here? I typically have to guess incorrectly once before figuring out the order on some of these :grinning: